server/internal/infrastructure/gcp/options.go

```diff
@@ -0,0 +1,29 @@
+package gcp
+
+type taskRunnerOptions struct {
+	subscriberURL string
```

Comment:
OK! Since `CredFilePath` is also removed, we don't need to have functional option for now. Just removed with [remove functional options](https://github.com/reearth/reearth-cms/pull/127/commits/d2d4e58a0f2979e5a1916918c5f3882ac95f9974)